windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -33,7 +33,7 @@ endif::openshift-origin[]
 [id="wmco-5-1-0"]
 == Release notes for Red Hat Windows Machine Config Operator 5.1.0
 
-This release of the WMCO is now available with a bug fix and a few improvements. The components of the WMCO 5.1.0 is now available in link:https://access.redhat.com/errata/RHBA-2022:95425-01[RHBA-2022:95425-01].
+This release of the WMCO is now available with a bug fix and a few improvements. The components of the WMCO 5.1.0 is now available in link:https://errata.devel.redhat.com/advisory/95425[RHBA-2022:4989-01].
```

Comment:
This should not link to errata.devel.redhat.com.  It should be similar to the link for the 5.0.0 release which is to the customer portal.
```suggestion
This release of the WMCO is now available with a bug fix and a few improvements. The components of the WMCO 5.1.0 is now available in link:https://access.redhat.com/errata/RHBA-2022:4989[RHBA-2022:4989].
```

Reply:
Yes correct